python-threatexchange/threatexchange/extensions/vpdq/tests/test_vpdq_hash_and_match.py

```diff
@@ -29,7 +29,7 @@
 
 @pytest.mark.skipif(_DISABLED, reason="vpdq not installed")
 def test_vpdq_from_string_path():
-    computed_hash = VPDQSignal.hash_from_file(ROOTDIR / VIDEO)
+    computed_hash = VPDQSignal.hash_from_file(ROOTDIR / VIDEO, seconds_per_hash=0)
```

Comment:
Could we update `test_hash.txt` instead of overriding the default?

Reply:
For test, it will be better if we test on every frame as we do not care about the efficiency but accuracy.

Reply:
I would echo my point in #1163 to have test match production behavior.

In this case the idea of having a test that is even more granular e.g. efficiency > accuracy would be nice to have as well

Could we possibly have both (i.e. two tests/test files)?

Reply:
I think I will keep the vpdq test for accuracy but change the tests in pytx to match the prod parameter. The vpdq test will only trigger when the actual hashing code has changed.